plugins/angular.js

```diff
@@ -65,3 +86,5 @@ angularPlugin._normalizeData = function (data) {
 };
 
 module.exports = angularPlugin;
+// Install plugin when included
+angularPlugin(window.Raven);
```

Comment:
By including this here, it makes testing particularly hard. The tests should be able to import the methods here and test them without side effects.